app/Config/Auth.php

```diff
@@ -224,7 +271,7 @@ class Auth extends ShieldAuth
      */
     public array $validFields = [
         'email',
-        'username',
+        // 'username',
```

Comment:
Is this okay?

Reply:
Currently yes, no problem. We currently have email login.
Probably originally came from Benfire.